content/6_Advanced/Game_Theory.mdx

```diff
@@ -2,35 +2,67 @@
 id: game-theory
 title: 'Game Theory'
 author: Benjamin Qi
-description: '?'
+contributors: Salma J
+description: 'Problems involving Algorithmic Game Theory'
```

Comment:
```suggestion
description: 'Problems involving Algorithmic Game Theory'
```

well that's the name of the module.
I'd suggest something like "deducing optimal strategies" or something

Reply:
Ok!